src/components/insights/PulseCard.svelte

```diff
@@ -81,6 +81,8 @@ mixin rawHtml(variables)
     @include text('body-1');
     display: flex;
     color: var(--rhino);
+    font-size: 24px;
+    line-height: 32px;
```

Comment:
These are the styles for the `h4` element in our [typography library](https://www.figma.com/file/a7RcFp4wWVCBl744AUmYykRF/Design-Library?node-id=7158%3A4329).
`@include text('body-1');` on line 81 should be changed to `@include text('h4');`